### ISSUE TITLE
When starting a taskа fatal error during event usage. error=a bytes-like object is required  

### ISSUE THREAD

**Issue by maltcev0i** [cert-ee/cuckoo3#18]
Hello, when I run a task after the test time has expired, I get the following error:

```
ERROR [cuckoo.processing.worker]: Fatal error during event usage. error=a bytes-like object is required 
Traceback (most recent call last):
  File "/opt/cuckoo3/processing/cuckoo/processing/worker.py", line 388, in start
    self._read_events()
  File "/opt/cuckoo3/processing/cuckoo/processing/worker.py", line 384, in _read_events
    consumer.use_event(event)
  File "/opt/cuckoo3/processing/cuckoo/processing/post/eventconsumer/patternsigs.py", line 92, in use_event
    event.pattern_scan(self.scanner, self.taskctx)
  File "/opt/cuckoo3/processing/cuckoo/processing/event/events.py", line 160, in pattern_scan
    pattern_scanner.scan(
  File "/opt/cuckoo3/processing/cuckoo/processing/signatures/pattern.py", line 1025, in scan
    scandb.scan(
TypeError: a bytes-like object is required
2022-10-19 08:33:40 ERROR [cuckoo.control]: Task post stage failed. task_id=20221019-SW3A0J_1

```

**Comment by lsilveirab39** [comment]
I had the same problem, I even reinstalled and the same problem occurred.

**Comment by amadisson** [comment]
Did you install the Cuckoo signatures?

**Comment by lsilveirab39** [comment]
Yes, i did it!

**Comment by Cryss76** [comment]
This was fixed with pull-request #23 .
please close @amadisson
